client/state/index.js

```diff
@@ -59,10 +60,12 @@ import ui from './ui/reducer';
 import users from './users/reducer';
 import wordads from './wordads/reducer';
 
+const log = debug( 'calypso:state:reducers' );
```

Comment:
just for your information, usually we're importing something like `debugFactory` in Calypso and then creating `const debug = debugFactory( … )`